x-pack/plugins/maps/public/shared/layers/sources/es_search_source/es_search_source.js

```diff
@@ -252,11 +335,31 @@ export class ESSearchSource extends AbstractESSource {
 
   getSourceTooltipContent(sourceDataRequest) {
     const featureCollection = sourceDataRequest ? sourceDataRequest.getData() : null;
-    const meta = sourceDataRequest ? sourceDataRequest.getMeta() : {};
+    const meta = sourceDataRequest ? sourceDataRequest.getMeta() : null;
+    if (!featureCollection || !meta) {
+      // no tooltip content needed when there is no feature collection or meta
+      return null;
+    }
+
+    if (this._isTopHits()) {
+      const entitiesFoundMsg = i18n.translate('xpack.maps.esSearch.topHitsEntitiesCountMsg', {
+        defaultMessage: `Found {entityCount} entities.`,
+        values: { entityCount: meta.entityCount }
```

Comment:
Accessing `meta.entityCount` probably just depends on how you're handling `meta` above. If you're checking for empty above and returning then there might not be an issue

Reply:
meta will be defined if feature collection exists so this will not be a problem.